RadClass/RadClass.py

```diff
@@ -86,28 +92,43 @@ def queue_file(self):
 
         self.processor = ds.DataSet(self.labels)
         self.processor.init_database(self.filename, self.datapath)
-        # parameters for keeping track of progress through a file
 
+        # possible invalid user input scenarios
+        # user start time is great than the last possible data timestamp
+        if self.start_time is not None and self.start_time > self.processor.timestamps[-1]:
+            raise ValueError('User start time is larger than all timestamps.')
+        # user stop time is less than the first possible timestamp
+        elif self.stop_time is not None and self.stop_time < self.processor.timestamps[0]:
+                raise ValueError('User stop time is less than the first timestamp.')
+        # user start time is greater than user stop time
+        elif self.start_time is not None and self.stop_time is not None and self.start_time > self.stop_time:
+            raise ValueError('User start time is larger than stop time.')
+
+        # parameters for keeping track of progress through a file
         self.start_i = 0
         self.stop_i = len(self.processor.timestamps) - 1
 
         if self.start_time is not None:
-            timestamp = self.processor.timestamps[self.processor.timestamps >=
-                                                  self.start_time][0]
             self.start_i = max(np.searchsorted(self.processor.timestamps,
-                                               timestamp,
+                                               self.start_time,
                                                side='right')-1, 0)
```

Comment:
```suggestion
                                               side='left'), 0)
```
Should this be left sided instead? I think this means it will pick the first timestamp >= self.start_time, so that no data before the user start_time is used.